datalake/src/main/java/com/sequenceiq/datalake/service/sdx/status/SdxStatusService.java

```diff
@@ -81,7 +81,7 @@ public SdxCluster setStatusForDatalakeAndNotify(DatalakeStatusEnum status, Resou
         }).orElseThrow(() -> new NotFoundException("SdxCluster was not found with ID: " + datalakeId));
     }
 
-    private void setStatusForDatalake(DatalakeStatusEnum status, String statusReason, SdxCluster sdxCluster) {
+    public void setStatusForDatalake(DatalakeStatusEnum status, String statusReason, SdxCluster sdxCluster) {
```

Comment:
SdxStatusService exposes API to update the status and notify the status to notify that status so that it can be rendered in the UI.

What I need in this case is an API that just updates the status and does not notify.

Till now there was no such need so setStatusForDatalake as a private method made sence. Now that that is need for an API why not make it a public API?